eng/build.yml

```diff
@@ -1,53 +1,54 @@
 parameters:
+  # Agent OS identifier and used as job name
   agentOs: ''
-  queue: {}
+
+  # Agent pool
+  pool: {}
+
+  # Additional variables
   variables: {}
 
-phases:
-- template: /eng/common/templates/phases/base.yml
+  # Build strategy - matrix
+  strategy: {}
+
+  # Job timeout
+  timeoutInMinutes: 180
+
+jobs:
+- template: /eng/common/templates/job/job.yml
   parameters:
     name: ${{ parameters.agentOs }}
-    enableTelemetry: true
     enableMicrobuild: true
-    queue: ${{ parameters.queue }}
+    enablePublishBuildArtifacts: true
+    enablePublishBuildAssets: true
+    enablePublishTestResults: true
+    enableTelemetry: true
+    helixRepo: dotnet/sdk
+    pool: ${{ parameters.pool }}
+    ${{ if ne(parameters.strategy, '') }}:
+      strategy: ${{ parameters.strategy }}
     variables: 
-      ${{ insert }}: ${{ parameters.variables }}
-      _AgentOSName: ${{ parameters.agentOs }}
-      _HelixType: build/product
-      _HelixBuildConfig: $(_BuildConfig)
-      _TeamName: DotNetCore
-      _PerformanceArgs: /p:PerfIterations=4
-      ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        _HelixSource: pr/dotnet/sdk/$(Build.SourceBranch)
-        _OfficialBuildIdArgs: ''
-        _PublishArgs: ''
-        _SignArgs: ''
-      ${{ if ne(variables['System.TeamProject'], 'public') }}:
-        _HelixSource: official/dotnet/sdk/$(Build.SourceBranch)
-        _OfficialBuildIdArgs: /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-        _PublishArgs: /p:DotNetPublishBlobFeedKey=$(dotnetfeed-storage-access-key-1)
-                  /p:DotNetPublishBlobFeedUrl=$(PB_PublishBlobFeedUrl)
-                  /p:DotNetPublishToBlobFeed=$(_DotNetPublishToBlobFeed)
-                  /p:DotNetSymbolServerTokenMsdl=$(microsoft-symbol-server-pat)
-                  /p:DotNetSymbolServerTokenSymWeb=$(symweb-symbol-server-pat)
-                  /p:PB_PublishType=$(_PublishType)
-        _SignArgs: /p:DotNetSignType=$(_SignType) /p:TeamName=$(_TeamName)
+      - ${{ insert }}: ${{ parameters.variables }}
```

Comment:
I *think* that this might work, but only because you only have one variable defined in `.vsts-ci.yml`.  As soon as you add another variable, I think yaml will get confused because you'll be trying to insert the yaml variables "object" into the variables "array" and it won't know what to do with that.  The way to fix this would be to change `.vsts-ci.yml` to pass variables as an array...

```yaml

      variables:
      - _PREVIEW_VSTS_DOCKER_IMAGE: microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-e435274-20180628134544
```
… and then insert each of those items here like we do in job.yml...

https://github.com/dotnet/arcade/blob/master/eng/common/templates/job/job.yml#L89-L109